src/components/DropZone/DropZone.tsx

```diff
@@ -177,6 +177,9 @@ export class DropZone extends React.Component<CombinedProps, State> {
       errorOverlayText: translate(`Polaris.DropZone.errorOverlayText${suffix}`),
       numFiles: 0,
     };
+    this.node = React.createRef();
```

Comment:
Rather than creating the refs in the constructor we can do it above

Reply:
Will move the refs above!